src/tests/example.spec.js

```diff
@@ -24,4 +23,45 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    test('Filter products on Inventory page', async ({ inventoryPage }) => {
+        await inventoryPage.filterItemsBy('Price (low to high)');
+        expect(await inventoryPage.verifyItemsSortedBy('Price (low to high)'), 'items should be sorted by price ASC').toBeTruthy();
+
+        await inventoryPage.filterItemsBy('Price (high to low)');
+        expect(await inventoryPage.verifyItemsSortedBy('Price (high to low)'), 'items should be sorted by price DESC').toBeTruthy();
+
+        await inventoryPage.filterItemsBy('Name (A to Z)');
+        expect(await inventoryPage.verifyItemsSortedBy('Name (A to Z)'), 'items should be sorted by name ASC').toBeTruthy();
+
+        await inventoryPage.filterItemsBy('Name (Z to A)');
+        expect(await inventoryPage.verifyItemsSortedBy('Name (Z to A)'), 'items should be sorted by name DESC').toBeTruthy();
+    });
```

Comment:
this it could be parametrized. Could we run it for the array of test data? To have separate tests for each sorting. if first sorting fail will you know are the another sorting work correctly?